docs/05-StatefulReactView.mdx

```diff
@@ -0,0 +1,167 @@
+---
+name: StatefulReactView
+route: /stateful-react-view
+---
+
+# StatefulReactView
+
+StatefulReactView is a higher order function (HOF) that ships with Brigade since 1.0.4.
```

Comment:
Little thing, but lets ship this with `1.1.0`. We should do a minor release for this as its adding a new feature.

Reply:
Great point! Also, I'm not sure how we should update the version number - should we do it in the PR itself, or using an `npm publish minor` or what?

Reply:
We can update the number here, but you can leave it in package.json. It'll get updated when we publish.